src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -212,6 +212,14 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                         </label>
                     </div>
                 </div>
+                <div class="form-group-narrow" *ngIf="!isInProduction()">
+                    <div class="form-check">
+                        <label class="form-check-label" for="field_noLocalSetup">
+                            <input class="form-check-input" type="checkbox" name="noLocalSetup" id="field_noLocalSetup" [(ngModel)]="programmingExercise.noLocalSetup" checked />
+                            <span jhiTranslate="artemisApp.programmingExercise.noLocalSetup">No Local Setup</span>
```

Comment:
I don't like the term "no local setup"!!!

a better term would be "No version control and continuous integration server available"

Reply:
i changed the name